<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACSegmentationControlsWidgetQt</class>
 <widget class="QWidget" name="ACSegmentationControlsWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>250</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Segmentation Controls</string>
  </property>
  <widget class="QGroupBox" name="groupBoxPlugins">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>64</height>
    </size>
   </property>
   <property name="title">
    <string>Plugins</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QComboBox" name="comboBoxPlugins">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxParameters">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>250</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>500</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>96</height>
    </size>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidgetParameters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>500</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>250</width>
      <height>32</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
